packages/sdk/src/types/Events.ts

```diff
@@ -0,0 +1,23 @@
+export interface EventsParams {
+	event?: string
+	customer?: {
+		source_id?: string
+		id?: string
+		name?: string
+		email?: string
+		metadata?: Record<string, any>
+	}
+	metadata?: Record<string, any>
+	referral?: {
+		code?: string
+		referrer_id?: string
+	}
+	loyalty?: {
+		code?: string
```

Comment:
The same it would be wise to use make `code` requried if you already using `loyalty`